TravelerApp/TravelerAppCore/Controller/SearchByLocalisation.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
w kodzie powinien być porządek i nie powinny znajdować się linijki które nie są potrzebne. Dlatego usuń wszystkie zbędne linie jak np nie używane usingi :) zastosuj do wszystkich plików gdzie to jest potrzebne :)

Reply:
Usunąłem niepotrzebne usingi oraz zbędne linie